Add locally built docker compose & docker usage instructions (#25)
ItsDrike/code-jam-2024: README.md

Finally, you can start the bot with `python -m src`.

> ### Using docker

Reviewer: Might be beneficial to have some example commands.

Finally, you can start the bot with `python -m src`.

### Using docker

The project also supports [docker](https://www.docker.com/) installation, which should allow running the project
anywhere, without installing all of the dependencies manually. This is a lot more convenient way to run the bot, if you
just want to run it and you don't wish to do any actual development.

> To use docker, you can check out the images that are automatically built after each update to the `main` branch on

Reviewer: ```suggestion
To use docker, you can check out the images that are automatically built after each update to the `main` branch at
```

[ghcr](https://github.com/itsdrike/code-jam-2024/pkgs/container/code-jam-2024). You can also use [`docker
compose`](https://docs.docker.com/compose/) with the [`docker-compose.yaml`](./docker-compose.yaml) file, which will
pull this image from ghcr. To run the container using this file, you can use the following command:

```bash
docker compose up
```

> 

Reviewer: ```suggestion

> [!NOTE]  
> To run the container in the background, add the `-d` flag to the command.

```

If you want to build the image locally (to include some other changes that aren't yet in the main branch, maybe during
development or to customize something when deploying), you can also use the

> [`docker-compose.local.yaml`](./docker-compose.local.yaml), which defines an image building step from our

Reviewer: ```suggestion
[`docker-compose.local.yaml`](./docker-compose.local.yaml) file, which defines an image building step from our
```

Reviewer: Alternatively remove "the" before the hyperlink.

[`Dockerfile`](./Dockerfile). To run this local version of docker-compose, you can use the following command:

```bash
docker compose -f ./docker-compose.local.yaml up
```

> Note that you will still need to create a `.env` file with all of the configuration variables (see [the configuring

Reviewer: ```suggestion
> [!IMPORTANT]  
> You will still need to create a `.env` file with all of the configuration variables (see [the configuring section](#configuring-the-bot).
```
